test/views/components/GovukFooterSpec.scala

```diff
@@ -35,17 +35,11 @@ class GovukFooterSpec extends ViewSpec with ViewFns with GuiceOneAppPerSuite {
 
   trait Test {
     private def injector: Injector = app.injector
-
     def messagesApi: MessagesApi = injector.instanceOf[MessagesApi]
     val footerLinks = app.injector.instanceOf[FooterLinks]
     val footer = app.injector.instanceOf[views.html.components.govukFooter]
     implicit def messages: Messages = messagesApi.preferred(Seq(Lang("en")))
```

Comment:
How do the Welsh tests work as I can't see an implicit Welsh declaration anywhere?

Reply:
Welsh tests removed as here there is only text drawn from message files and the language functioning is tested elsewhere